miio/airhumidifier_jsqs.py

```diff
@@ -0,0 +1,249 @@
+    Low = 1
+    Mid = 2
+    High = 3
+    Auto = 4
+
+
+class AirHumidifierJsqsStatus(DeviceStatus):
+    """Container for status reports from the air humidifier.
+
+    Xiaomi Mi Smart Humidifer S (deerma.humidifier.jsqs) respone (MIoT format)
+
+    [
+        {'did': 'power', 'siid': 2, 'piid': 1, 'code': 0, 'value': True},
+        {'did': 'fault', 'siid': 2, 'piid': 2, 'code': 0, 'value': 0},
+        {'did': 'mode', 'siid': 2, 'piid': 5, 'code': 0, 'value': 1},
+        {'did': 'target_humidity', 'siid': 2, 'piid': 6, 'code': 0, 'value': 50},
+        {'did': 'relative_humidity', 'siid': 3, 'piid': 1, 'code': 0, 'value': 40},
+        {'did': 'temperature', 'siid': 3, 'piid': 7, 'code': 0, 'value': 22.7},
+        {'did': 'buzzer', 'siid': 5, 'piid': 1, 'code': 0, 'value': False},
+        {'did': 'led_light', 'siid': 6, 'piid': 1, 'code': 0, 'value': True},
+        {'did': 'water_shortage_fault', 'siid': 7, 'piid': 1, 'code': 0, 'value': False},
+        {'did': 'tank_filed', 'siid': 7, 'piid': 2, 'code': 0, 'value': False},
+        {'did': 'overwet_protect', 'siid': 7, 'piid': 3, 'code': 0, 'value': False}
+    ]
+    """
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.data = data
+
+    # Air Humidifier
+
+    @property
+    def is_on(self) -> bool:
+        """Return True if device is on."""
+        return self.data["power"]
+
+    @property
+    def power(self) -> str:
+        """Return power state."""
+        return "on" if self.is_on else "off"
+
+    @property
+    def error(self) -> int:
+        """Return error state."""
+        return self.data["fault"]
+
+    @property
+    def mode(self) -> OperationMode:
+        """Return current operation mode."""
+
+        try:
+            mode = OperationMode(self.data["mode"])
+        except ValueError as e:
+            _LOGGER.exception("Cannot parse mode: %s", e)
+            return OperationMode.Auto
+
+        return mode
+
+    @property
+    def target_humidity(self) -> int:
+        """Return target humidity."""
+        return self.data["target_humidity"]
+
+    # Environment
+
+    @property
+    def relative_humidity(self) -> int:
+        """Return current humidity."""
+        return self.data["relative_humidity"]
+
+    @property
+    def temperature(self) -> Optional[float]:
+        """Return current temperature, if available."""
+        if self.data["temperature"] is not None:
+            return round(self.data["temperature"], 1)
+        return None
+
+    # Alarm
+
+    @property
+    def buzzer(self) -> Optional[bool]:
+        """Return True if buzzer is on."""
+        if self.data["buzzer"] is not None:
+            return self.data["buzzer"]
+        return None
+
+    # Indicator Light
+
+    @property
+    def led_light(self) -> Optional[bool]:
+        """Return status of the LED."""
+
+        if self.data["led_light"] is not None:
+            return self.data["led_light"]
+        return None
```

Comment:
You can replace all similar structures with this:
```suggestion
        return self.data.get("led_light")
```